src/main/webapp/scripts/controller-auth.js

```diff
@@ -0,0 +1,61 @@
+import { postVideoComment } from '/scripts/post-comment.js';
+import { videoId, commentElement, failedInitCallback, successfulApiCallback, failedApiCallback, updateSigninStatus } from '/scripts/view.js';
+
+/** API key provied by GCP to authenticate the client*/
+const apiKey = '';
+/** client ID provied by GCP to identify the client*/
```

Comment:
Nit: please leave a space before `*/` here and elsewhere i.e. `client */`

Reply:
Yap done